src/service/kb_auth.py

```diff
@@ -16,3 +24,89 @@ class AdminPermission(IntEnum):
     # leave some space for potential future levels
     FULL = 10
 
+
+class KBaseAuth:
+    ''' A client for contacting the KBase authentication server. '''
+
+    def __init__(
+            self,
+            auth_url: str,
+            full_admin_roles: List[str] = None,
```

Comment:
shouldn't we make the default `full_admin_roles` to some fixed set just in case someone inits this class with a wrong `full_admin_roles` list?

Reply:
Not totally sure I understand what you're asking, but `full_admin_roles` should be configurable, so they need to be an input arg.

As to why they need to be configurable, we may wish to rename roles, or add new roles that work for several (but not all services). If they're hardcoded we can't do that.

I'm going to go ahead and merge since you approved, but if you want to keep talking about it we can do it here or in slack and make any changes we come up with in a future PR

Reply:
Another option is just returning the roles and letting the caller figure out what do to with them I guess